README.md

```diff
@@ -1,26 +1,46 @@
 # token-supply
+
 REST API for CoinGecko and CoinMarketCap
 
-## Run the server locally
+## Setup
 
 Install the dependencies:
 ```
+yarn global add vercel
 yarn install
 ```
 
-Add variables to your local development environment:
+Create the project:
+```
+vercel
```

Comment:
@tnrdd When I type `vercel`, I get "No existing credentials found":
```

vercel
Vercel CLI 28.0.1
> > No existing credentials found. Please log in:
> Log in to Vercel (Use arrow keys)
❯ Continue with GitHub 
  Continue with GitLab 
  Continue with Bitbucket 
  Continue with Email 
  Continue with SAML Single Sign-On 
  ─────────────────────────────────
  Abort 
```

Is this a necessary step for developing the code locally?

Reply:
Yes need a Vercel account to run it locally too.